### DISCUSSION TITLE
How to shift Github Student Developer Pack to other GitHub account ?

### DISCUSSION THREAD

**Discussion by sagar-io** [community/community#49875] [GitHub Education] [answered]
I have GitHub Student Developer Pack that is associated with my college email address and the account associated with my college email address doesn't have any contributions in GitHub. On the other hand, I've another GitHub account associated with my personal email address that I've been using for a long time and it has many repo and contributions. 
I'm facing issues while applying for Octernships in GitHub that requires a Developer pack and several contributions but I'm not eligible because the account that has a Developer pack doesn't have contributions.

**Comment by samus-aran** [comment, marked as answer]
Hi @sagar-io  

It’s not possible to transfer Global Campus benefits over from one GitHub account to another.

However, you are able to merge your accounts, but the account with the benefits would be the main account. You can add your non-academic email from the previous account to the account containing the academic email. If you desire you can then rename this account once merged. For more details check out out the [merging of your personal accounts](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/merging-multiple-personal-accounts) documentation.

**Reply by drewsix9** [reply]
In his case, can his personal account use the github student developer pack if he does merge his accounts?

**Comment by danidre14** [comment]
So unfortunate. I find myself in this same predicament.

Have my personal/professional official email since 2014. Sometime last year saw Global Campus and created a student account to do it, but for projects and continued professional work went back to this official account.

Now, just watched [GitHub GitHub Education: free programs, technology, and opportunities available for Students](https://www.youtube.com/watch?v=HIVFdN9VGgw) and can't switch to this account. It's too difficult to transfer everything and merge this into the student account, because of many repositories, organizations, and links that will break.

But I can't transfer that student account to this one. So now I have to live two lives. And hope I can be successful at the Octernships if it is true that you need contributions... (maybe I can link this official account)

**Reply by vedantmishra03** [reply]
what happed if you add student account to main account does all the offers are applicable on the same github ??????

**Reply by vedantmishra03** [reply]
same means the older one??
